chains/manager.go

```diff
@@ -201,8 +199,7 @@ type ManagerConfig struct {
 	CriticalChains            set.Set[ids.ID] // Chains that can't exit gracefully
 	TimeoutManager            timeout.Manager // Manages request timeouts when sending messages to other validators
 	Health                    health.Registerer
-	SubnetConfigs             map[ids.ID]subnets.Config // ID -> SubnetConfig
```

Comment:
Where did we move those?

Reply:
I originally thought we could remove this since we're only using the subnet configs for their proposervm configs and this PR changes the proposervm to be an opt-in feature that vm implementations wrap themselves in (for example hypersdk does not want to use the proposervm). In the next iteration we might need to keep this since the X-Chain only wants to use the proposervm once it linearizes... we can't just blindly return a wrapped AVM instance in a proposervm since proposervm is not compatible pre-linearization.

Reply:
There are bunch of other configs in subnet configs https://github.com/ava-labs/avalanchego/pull/2807/files#diff-7a895c6b0e074cfed9bbd7a1bf256218f85323f62f5fff563b82eba35f34d58dR27-R37

AFAIK AvaCloud uses validator only config https://support.avacloud.io/will-my-network-be-private

Reply:
I think there might be some confusion, this PR does not remove the usage of subnet configs. The subnet configs are still used [here](https://github.com/ava-labs/avalanchego/blob/f1f88ac2775aeeb790b5463747be5850bcd78419/chains/subnets.go#L20), we just have a copy of these configs that we don't need

Reply:
aahh sorry my bad, I wasn't aware that we already moved them. Then it's looking good 👍